src/dolphin/main.ts

```diff
@@ -115,10 +115,10 @@ ipc_checkDesktopAppDolphin.main!.handle(async () => {
   return { dolphinPath: dolphinExecutablePath, exists: exists };
 });
 
-dolphinManager.on("playback-dolphin-closed", async () => {
-  void ipc_dolphinClosedEvent.main!.trigger({ dolphinType: DolphinLaunchType.PLAYBACK });
+dolphinManager.on("playback-dolphin-closed", async (playbackId: string, code: number) => {
```

Comment:
```suggestion
dolphinManager.on("playback-dolphin-closed", async (_playbackId: string, code: number) => {
```
since it is unused

Reply:
Cool thanks, I was gonna ask how to do this.